packages/analytics/src/Analytics.ts

```diff
@@ -687,6 +701,7 @@ class Analytics {
       event: {
         ...eventContext,
         [ANALYTICS_UNIQUE_EVENT_ID]: uuid(),
+        [ANALYTICS_PACKAGE_VERSION]: this.analyticsVersion,
```

Comment:
Don't we have a library property in the event data already that contains this info? Why should it be specific to the event?